scripts/camViewer

```diff
@@ -152,29 +154,17 @@ if [[ ($1 == "--help") || ($1 == "-h") ]]; then
 	exit 0
 fi
 
-hutch=`get_info --gethutch`
-if [ $hutch = unknown_hutch ]; then
-    echo 'unknown hutch, cannot use camViewer script'
-    exit 1
-fi
-
-CAMNUM=0
-WAIT=0
-RATE=-1
-MAINSCREEN=0
 
-EXE=/reg/g/pcds/pyps/config/$hutch/camviewer/run_viewer.csh
-PVLIST=/reg/g/pcds/pyps/config/$hutch/camviewer.cfg
-imgr=/reg/g/pcds/pyps/apps/iocmanager/latest/imgr
 CAMNUM=0
 WAIT=0
 RATE=-1
 REBOOT=0
 ENABLE=0
 DISABLE=0
+MAINSCREEN=0
 
 
-while getopts "c:w:u:H:lmred" OPTION
+while getopts "c:w:u:H:lmred":"-enab":"-disab" OPTION
```

Comment:
Does that work for getopts? If not, it might be easier to get rid of getopts and loop through the arguments the old-fashioned way

Reply:
Is actually resolved?

Reply:
Yes.